src/components/data-processing/StatusIndicator.jsx

```diff
@@ -112,24 +113,57 @@ const StatusIndicator = (props) => {
     },
   };
 
-  const renderOverlay = () => (
-    <Card style={{ width: 300, padding: 16 }}>
-      <Paragraph>
-        <Text strong>
-          Your data processing is
-          {' '}
-          {statusIndicators[status].title}
-          .
-        </Text>
-      </Paragraph>
-      <Paragraph>
-        <Text>{`${completedSteps.length} of ${allSteps.length} steps complete`}</Text>
-      </Paragraph>
-      <Paragraph>
-        {statusIndicators[status].description}
-      </Paragraph>
-    </Card>
-  );
+  const renderOverlay = () => {
+    const renderOverlayContent = () => {
+      if (loadingBackendStatus) { return (<Text>Loading run status...</Text>); }
+      if (errorLoadingBackendStatus) {
```

Comment:
not sure how to test this in the staging like

Reply:
It can't be tested in staging - we have to setup our own conditions in the UI